SA3/qa_web/models.py

```diff
@@ -1,7 +1,14 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 
 class Post(models.Model):
     content = models.TextField()
-    owner = models.CharField(max_length=30)
+    owner = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
```

Comment:
A really good modification. It will be better if there is a comment explaining parameter's use.